libs/components/datetime/src/lib/modules/datepicker/datepicker-input.directive.ts

```diff
@@ -332,7 +332,7 @@ export class SkyDatepickerInputDirective
 
     this.#control?.setErrors({
       skyDate: {
-        invalid: true,
+        invalid: value,
```

Comment:
Mind replying with a reason this change was done in case we come back to this pull request in the future?

Reply:
reason: this change is to bring this branch of error handling logic in line with all other instances of the datepicker directive setting `invalid` error.